src/Apps/CollectorProfile/Routes/Saves/Components/CreateNewListModal/__tests__/CreateNewListModal.jest.tsx

```diff
@@ -199,4 +204,62 @@ describe("CreateNewListModal", () => {
       expect(screen.getByText("Name cannot be empty")).toBeInTheDocument()
     })
   })
+
+  describe("shared list toggle", () => {
+    beforeEach(() => {
+      ;(useFeatureFlag as jest.Mock).mockImplementation(() => true)
```

Comment:
Another minor one that's a good habit so one doesn't need to keep casting, placed at the top somewhere (these tend to get copied around and duplicated):
```tsx
const mockUseFeatureFlag = useFeatureFlag as jest.Mock
```